Source/Core/DolphinQt/TAS/WiiTASInputWindow.cpp

```diff
@@ -268,6 +220,125 @@ WiiTASInputWindow::WiiTASInputWindow(QWidget* parent, int num) : TASInputWindow(
   UpdateExt(ext);
 }
 
+void WiiTASInputWindow::SetOrientationAndTriggerBoxes()
+{
+  m_remote_orientation_box = new QGroupBox(tr("Wii Remote Orientation"));
+
+  const QKeySequence remote_x_shortcut_key_sequence = QKeySequence(Qt::ALT + Qt::Key_Q);
+  const QKeySequence remote_y_shortcut_key_sequence = QKeySequence(Qt::ALT + Qt::Key_W);
+  const QKeySequence remote_z_shortcut_key_sequence = QKeySequence(Qt::ALT + Qt::Key_E);
+
+  auto* remote_x_label = new QLabel(QStringLiteral("%1 (%2").arg(
```

Comment:
Ditto, with regards to the string translation comment made in a previous comment